crates/weaver_forge/src/lib.rs

```diff
@@ -430,16 +440,36 @@ impl TemplateEngine {
 
         let syntax = SyntaxConfig::builder()
             .block_delimiters(
-                Cow::Owned(template_syntax.block_start),
-                Cow::Owned(template_syntax.block_end),
+                Cow::Owned(
+                    template_syntax
+                        .block_start
+                        .unwrap_or_else(|| "{%".to_owned()),
+                ),
+                Cow::Owned(template_syntax.block_end.unwrap_or_else(|| "%}".to_owned())),
             )
             .variable_delimiters(
-                Cow::Owned(template_syntax.variable_start),
-                Cow::Owned(template_syntax.variable_end),
+                Cow::Owned(
+                    template_syntax
+                        .variable_start
+                        .unwrap_or_else(|| "{{".to_owned()),
```

Comment:
Nit - We seem to have lost a simple-to-find place for defaults.  Can we at least make these constants in the header of this file?

Reply:
Fixed in [78d4f40](https://github.com/open-telemetry/weaver/pull/231/commits/78d4f40d3ab653a85cc557ffa41876aff0866c73)